src/nft/components/collection/CollectionStats.tsx

```diff
@@ -251,22 +273,22 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
       <StatsItem label="Items" isMobile={isMobile ?? false}>
         {totalSupplyStr}
       </StatsItem>
-      {numOwnersStr ? (
+      {numOwnersStr || isCollectionStatsLoading ? (
```

Comment:
similar comments as above regardling doing 1 check for isCollectionStatsLoading and mapping an array to X number of divs